ladder/teams/kvstore.yaml

```diff
@@ -1,6 +1,4 @@
 members:
 - aanm
-- fristonio
 - giorio94
 - marseel
-- nirmoy
```

Comment:
fyi @nirmoy if you'd like to remain as a reviewer, please reach out.

Reply:
+1 from me, unfortunately I can't contribute to this repo at this moment.